### DISCUSSION TITLE
how to get the url

### DISCUSSION THREAD

**Discussion by hillshome** [robbrad/UKBinCollectionData#604] [Q&A] [answered]
can anyone tell me how i get the url of the remote Selenium for the integration? Thanks

**Comment by dp247** [comment, marked as answer]
You have to host your own instance on whatever hardware you're using.

So like for me, I have a docker container for standalone-chrome running on my Unraid server.

**Reply by hillshome** [reply]
How's is this achieved? I run home assistant on raspberry pi.

**Reply by dp247** [reply]
I'd hazard a guess that you could install docker and then pull the [standalone-chrome](https://hub.docker.com/r/selenium/standalone-chrome) image. I haven't tried though, so may need a bit of tinkering :)

**Reply by Alllfff** [reply]
@dp247 did you manage? Thanks

**Reply by dp247** [reply]
> @dp247 did you manage? Thanks

Unfortunately not, I haven't had time for this project recently.
